### DISCUSSION TITLE
Help! even if glass color is set to 858585 menu still has no dark shade

### DISCUSSION THREAD

**Discussion by ghost** [bonzibudd/Fluent-Metro#165] [General]
hi,
so i set the glass color to #858585 but the menu still doesn't have the dark shade

**Comment by bonzibudd** [comment]
Fluent-Metro uses your system color by default, so if you set your Windows color to dark, it will follow that. If that doesn’t work, make sure you are on Open-Shell version 4.4.160 or later. Otherwise, you can override the colors in Open-Shell Menu settings under the `Skin` tab.
